industrial_ci/rosdep-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
Not sure how to properly comment a shebang..
Perhaps we should set it explicitly as in https://github.com/ros-industrial/industrial_ci/blob/master/industrial_ci/ci_main.sh#L38